### PR TITLE
[IN PROGRESS] resolution of comments from Scott in PR #114.

### PR DESCRIPTION
**Pull request by jooolia**
Pull request to address comments given by Scott in PR #51 

- [x] docs are automatically created now for all ropensci pkgs, you can link to https://docs.ropensci.org/rsnps/ (it's built from the default branch (master) so doesn't reflect your work on dev branch)
- [x] I'd use remotes instead of devtools in the README for installation

- [x] would be good to make all http links in the README https (those that can be) so that there are as few insecure links as possible in the docs page https://docs.ropensci.org/rsnps
- [x] there's some exported functions in utils.R, are those functions meant to be used by package users? swap/flip/split_to_df. They aren't in the unit tests, and they aren't used in other functions in the package.
- [x] It's a good idea to rename the vignette from rsnps_vignette.Rmd to rsnps.Rmd as the vignette will then show up as "Getting Started" on the docs page, e.g. https://docs.ropensci.org/crul/articles/crul
- [x] the output of ncbi_snp_query: column names are a mix of all lowercase, or first letter uppercase, and some TitleCase. this seems like a good time to standardize them all to one style
	- [x] fix associated tests to expect new names
- [x] i dont think the XML or xml2 packages are used in rsnps anymore, i'd remove them if not
- [x] probably want to update the rsnps-package.R file 
	- [x] with the current authors (you two), 
	- [x] and update the info in that file, e.g, it still references ncbi_snp_query2 function
- [x] before submitting to cran: make sure to update codemeta.json codemetar::write_codemeta()

### CODE REVIEW

**Comment by jooolia** [comment]
Hi @sinarueeger,
Here is a branch that we can use to address the nice comments from Scott. I will update it as I work away on things.

File: R/ncbi_snp_api.R [hunk 1]
```diff
@@ -128,19 +128,19 @@ get_gene_names <- function(primary_info) {
...
 #' 
 #' This function currently pulling data for Assembly 38 - in particular
 #' note that if you think the BP position is wrong, that you may be 
 #' hoping for the BP position for a different Assembly. 
 #'
 #' @export
-#' @param SNPs (character) A vector of SNPs (rs numbers).
-#' @param ... Curl options passed on to [crul::HttpClient]
+#' @param snps (character) A vector of SNPs (rs numbers).
+#' @param ... Curl options passed on to [httr:GET]
```
Comment (jooolia) [inline R/ncbi_snp_api.R:143]: oops this will be fixed in the next commit.

**Comment by jooolia** [comment]
Hi @sinarueeger ,
I merged in the changes from your branch T114-sr to this branch so that we could be sure that all changes were done and play well together. Does this look good to you?

**Comment by sinarueeger** [comment]
Great, yes, lets merge!

**Pull request merged**